GOVERNANCE.md

```diff
@@ -0,0 +1,5 @@
+# Node.js GitHub Bot Project Governance
+
+The Node.js GitHub Bot is overseen by the Node.js Technical Steering Committee.
+https://github.com/nodejs/TSC
+
```

Comment:
The problem with teams is that they're not viewable to people that are not members of the org. In rare scenario there is someone that needs to escalate something beyond the bot's issue tracker- we should provide them a group with an open/transparent governance body.